### ISSUE TITLE
Unable to start new app on latest (0.56.0) - problems.ignore column does not exist

### ISSUE THREAD

**Issue by ADobin** [manyfold3d/manyfold#1821]
# Problem
I wanted to try out this project today with the `latest` docker image, which I matches the 0.56.0 version at the time of writing. After copying the `docker-compose.yml` from the website and running `docker compose up` the app container repeatedly fails to start on a failed database migration. The errors change slightly depending on how many times the container has crashed and tried to run the database migration but they all stem from `problems.ignore` not existing.

Below is one of the errors but on every startup there is a variation of this error.
```
== Schema =====================================================================

== 20230307215826 AddIconToLibrary: migrating =================================
rails aborted!
StandardError: An error has occurred, this and all later migrations canceled: (StandardError)

PG::UndefinedColumn: ERROR:  column problems.ignored does not exist
LINE 1: SELECT "problems".* FROM "problems" WHERE "problems"."ignore...
```

# Mitigation
For now, if I set version to `0.55.2` for the app image I'm able to start the application no problem. After getting it running on that version, I can switch to `latest`/`0.56.0` and the upgrade works fine.

**Comment by ADobin** [comment]
Ah, looks like while I was writing this #1820 was also filed for the same issue.

**Comment by Floppy** [comment]
Yeah, same issue, I'll close this as a duplicate. Should get a fix out soon.

**Comment by Floppy** [comment]
Fix released now in v0.56.1. Thanks for reporting ❤️
